File: FMAplication/Domain/DailyTasks/DailyAVTask.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;
using FMAplication.Core;
using FMAplication.Domain.AVCommunications;
using FMAplication.Domain.Bases;
using FMAplication.Domain.Sales;
using FMAplication.Enumerations;

namespace FMAplication.Domain.DailyTasks
{
    public class DailyAVTask : DailyBaseTask
    {

        public int AvSetupId { get; set; }
    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/DailyTasks/dailyAVTask.ts
import { Outlet } from "..";
import { AvSetup } from "../AVCommunications/avSetup";
import { Reason } from "../ExecutionReason/ExecutionReason";
import { DailyBaseTask } from "./dailyBaseTask";

export class DailyAVTask extends DailyBaseTask{
    id: number;
    avSetupId: number;
    avSetup:AvSetup;    
}